pkg/metrics/metrics.go

```diff
@@ -75,7 +81,7 @@ func PrometheusMiddleware(ph http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 
 		path, _ := mux.CurrentRoute(r).GetPathTemplate()
-		if path != handlers.LivezPath && path != handlers.ReadyzPath {
+		if path != "/livez" && path != "/readyz" {
```

Comment:
Any particular reason to stop referring to the constants here?

Reply:
It will cause the cycle import, where the metrics package imports the handlers package, and handlers package imports the metrics package at the same time. Which is not allowed in golang.

The only use of handlers in metrics.go is the name of those two paths.

Reply:
Ohhh, right, I get you. Maybe we can break out all the constants into a dedicated `consts` package like we do in ocm-agent-operator. https://github.com/openshift/ocm-agent-operator/tree/master/pkg/consts

Reply:
yeah, put all the consts into a dedicated package is a good idea